cluster/juju/charms/trusty/kubernetes-master/README.md

```diff
@@ -70,8 +70,8 @@ For more information on the recommended bundle deployment, see the
 #### Post Deployment
 
 To interact with the kubernetes environment, either build or
-[download](https://github.com/GoogleCloudPlatform/kubernetes/releases) the
-[kubectl](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+[download](https://github.com/kubernetes/kubernetes/releases) the
+[kubectl](https://github.com/kubernetes/kubernetes/blob/master/docs/kubectl.md)
```

Comment:
actually, links like these should probably be relative: ../../../../../docs/user-guide/kubectl/kubectl.md